sdks/python/setup.py

```diff
@@ -30,8 +30,7 @@
 
 def get_version():
   global_names = {}
-  execfile(os.path.normpath('./apache_beam/version.py'),
-           global_names)
+  exec(open(os.path.normpath('./apache_beam/version.py')).read(), global_names)
```

Comment:
The reasons was, this executes at setup time before `apache_beam` is installed, and `import` does not work at that stage.

If there is a way to `import` it instead of `exec` I would prefer that too.